core/modules/wmgr/python/dbMgr.py

```diff
@@ -810,7 +944,7 @@ def getIndex(dbName, tblName, chunkId=None):
             if not rows:
                 break
             for row in rows:
-                result.append(list(row))
+                result.append(tuple(row))
 
```

Comment:
That needs a new ticket. Note also that architecture is likely going to change in the future when we have data distribution mechanism in place which is supposed to take over some of the current functions of data loader.